scenarioo-server/src/main/java/org/scenarioo/business/diffViewer/comparator/AbstractComparator.java

```diff
@@ -36,21 +36,24 @@
 	protected static final ConfigurationRepository configurationRepository = RepositoryLocator.INSTANCE
 		.getConfigurationRepository();
 
+	protected ComparatorParameter parameter;
 	protected ScenarioDocuReader docuReader;
 	protected DiffWriter diffWriter;
 	protected String baseBranchName;
 	protected String baseBuildName;
 	protected ComparisonConfiguration comparisonConfiguration;
 
-	public AbstractComparator(String baseBranchName, String baseBuildName, final ComparisonConfiguration comparisonConfiguration) {
+	public AbstractComparator(ComparatorParameter parameter) {
+		this.parameter = parameter;
+		this.baseBranchName = parameter.getBaseBranchName();
+		this.baseBuildName = parameter.getBaseBuildName();
+		this.comparisonConfiguration = parameter.getComparisonConfiguration();
```

Comment:
hmm, I dont know, now the parameters are now somehow stored in two places ... once in `parameter` and once in the other fields of this class - this seems to me confusing.